src/System.Data.SqlClient/src/System/Data/SqlClient/SqlFileStream.Unsupported.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data.Common;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Globalization;
+using System.IO;
+using System.Diagnostics;
+using System.Runtime.Versioning;
+
+namespace System.Data.SqlTypes
+{
+    public sealed partial class SqlFileStream : System.IO.Stream
+    {
+        public SqlFileStream
+            (
```

Comment:
Commented elsewhere. Can you please pull the arguments into our normal styling? Also it looks like you have more usings than needed in this file.

Reply:
Updated to normal styling as requested.